projects/laji/src/app/shared-modules/download-request/download-request/download-request.component.html

```diff
@@ -14,7 +14,7 @@ <h4>{{ 'downloadRequest.download' | translate }}</h4>
         role="alert"
         class="alert alert-success"
         [innerHTML]='"downloadRequest.fileDownload.licenseReminder" | translate:{
-          filename: "<code>readme.txt</code>"
+          filename: "<code>" + ("downloadRequest.fileDownload.licenseFile" | translate) + "</code>"
```

Comment:
Could this be refactored to not use innerHTML
eg.
```html
<div ...>
  <code>
    {{ getLicenseReminderText$ | async }}
    (or using translate.instant without the async is fine too)
  </code>
</div>
```

InnerHTML behavior is pretty hard to predict... i'm surprised that pipes work within innerhtml at all actually